request_changes.py

```diff
@@ -0,0 +1,371 @@
+                    conn.commit()
+                    logging.info("Updated PR %s in %s to CHANGES REQUESTED.", pr_number, repo)
+                except psycopg2.Error as e:
+                    logging.error("Error updating database: %s", e)
+
+
+def extract_number_from_body(text):
+    try:
+        match = re.search(r"#\d+", str(text))
+        if match:
+            return int(match.group()[1:])
+    except ValueError as e:
+        logging.error("An error occurred while converting match group to int: %s", e)
+        return None
+    except re.error as e:
+        logging.error("An error occurred while searching text: %s", e)
+        return None
+    return None
+
+
+def update_squad_and_title(cur, conn, rtc, changes_tab):
+    logging.info("Updating squads and titles...")
+    try:
+        cur.execute(f"SELECT * FROM {changes_tab};")
+        open_issues_rows = cur.fetchall()
+
+        for row in open_issues_rows:
+            cur.execute(
+                f"""UPDATE {changes_tab}
+                    SET "Service Name" = rtc."Title", "Squad" = rtc."Squad"
+                    FROM {rtc} AS rtc
+                    WHERE {changes_tab}."Service Name" = rtc."Repository"
+                    AND {changes_tab}.id = %s;""",
+                (row[0],)
+            )
+            cur.execute(
+                f"""UPDATE {changes_tab}
+                    SET "Squad" = 'Other'
+                    WHERE {changes_tab}."Service Name" IN ('doc-exports', 'docs_on_docs', 'docsportal')
+                    AND {changes_tab}.id = %s;""",
+                (row[0],)
+            )
+        conn.commit()
+
+    except Exception as e:
+        logging.error("Error updating squad and title: %s", e)
+
+
+def main(org, rtc, changes_tab):
+    check_env_variables()
+
+    cur_csv.execute(f"DROP TABLE IF EXISTS {changes_tab}")
+    conn_csv.commit()
+
+    create_prs_table(conn_csv, cur_csv, changes_tab)
+
+    repos = get_repos(cur_csv, rtc)
+
+    logging.info("Gathering PRs where changes has been requested...")
+
+    for repo in repos:
+        prs = get_pr_number(org, repo)
+        for pr_info in prs:
+            pr_number = pr_info['pr_number']
+            process_pr_reviews(org, repo, pr_number, changes_tab)
+
+    parent_pr_changes_check(cur_csv, conn_csv, org, changes_tab)
+    parent_pr_changes_check(cur_csv, conn_csv, org, "our_side_problem")
+    update_squad_and_title(cur_csv, conn_csv, rtc, changes_tab)
+
+
+if __name__ == "__main__":
+    org_string = "docs"
+    rtc_table = "repo_title_category"
+    changes_table = "requested_changes"
+
+    conn_csv = connect_to_db(db_csv)
+    cur_csv = conn_csv.cursor()
+
+    done = False
+
+    cur_csv.execute("DROP TABLE IF EXISTS our_side_problem")
+    create_prs_table(conn_csv, cur_csv, "our_side_problem")
+
+    conn_csv.commit()
+
+    main(org_string, rtc_table, changes_table)
+    main(f"{org_string}-swiss", f"{rtc_table}_swiss", f"{changes_table}_swiss")
```

Comment:
use for cycle by dictionary with envs,  in case that in future there will be more environments and add logging for each call. Also main is a bad name for this func